### PR TITLE
chore: hermione -> testplane

### PR DESCRIPTION
**Pull request by KuznetsovRoman**

### CODE REVIEW

**Review by miripiruni — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "hermione-oauth",
-  "version": "2.0.0",
+  "name": "@testplane/oauth",
+  "version": "0.0.1",
```
Comment (sipayRT) [inline package.json:3]: why not `1.0.0`?

File: src/index.test.ts [hunk 2]
```diff
@@ -1,65 +1,68 @@
...
 
-describe("hermione-oauth", () => {
-    const browser = (config: object = {}): Hermione.BrowserConfig => config as unknown as Hermione.BrowserConfig;
+describe("@testplane/oauth", () => {
+    const browser = (config: object = {}): BrowserConfig => config as unknown as BrowserConfig;
 
-    const hermioneMock = (browsers: Record<string, Hermione.BrowserConfig>): Hermione => {
-        const emitter = new EventEmitter() as unknown as Hermione;
+    const testplaneMock = (browsers: Record<string, BrowserConfig>): Testplane => {
+        const emitter = new EventEmitter() as any;
```
Comment (sipayRT) [inline src/index.test.ts:20]: Why did you decide to add `ant` here instead of the old variant?

**Review by sipayRT — APPROVED** [review]

File: src/index.test.ts (hunk 2 above)
Comment (KuznetsovRoman) [reply src/index.test.ts:20]: Because later we are trying to set some properties of emitter. And we can't do that since hermione@8: those are readonly

Comment (KuznetsovRoman) [reply src/index.test.ts:20]: Because later we are trying to set some properties of emitter. And we can't do that since hermione@8: those are readonly

**Pull request merged**